examples/human/human.py

```diff
@@ -0,0 +1,10 @@
+from inspect_ai import Task, task
+from inspect_ai.solver._human_agent import human_agent
```

Comment:
It's an oversight that I didn't switch this example to `inspect_ai.agent.human_cli` -- could you do this as part of the PR?